vendor/darwin/Foundation/NSApplication.odin

```diff
@@ -12,98 +12,115 @@ ActivationPolicy :: enum UInteger {
-ApplicationPresentationOptionsDisableAppleMenu                :: ApplicationPresentationOptions {.DisableAppleMenu}
-ApplicationPresentationOptionsDisableProcessSwitching         :: ApplicationPresentationOptions {.DisableProcessSwitching}
-ApplicationPresentationOptionsDisableForceQuit                :: ApplicationPresentationOptions {.DisableForceQuit}
-ApplicationPresentationOptionsDisableSessionTermination       :: ApplicationPresentationOptions {.DisableSessionTermination}
-ApplicationPresentationOptionsDisableHideApplication          :: ApplicationPresentationOptions {.DisableHideApplication}
-ApplicationPresentationOptionsDisableMenuBarTransparency      :: ApplicationPresentationOptions {.DisableMenuBarTransparency}
-ApplicationPresentationOptionsFullScreen                      :: ApplicationPresentationOptions {.FullScreen}
-ApplicationPresentationOptionsAutoHideToolbar                 :: ApplicationPresentationOptions {.AutoHideToolbar}
-ApplicationPresentationOptionsDisableCursorLocationAssistance :: ApplicationPresentationOptions {.DisableCursorLocationAssistance}
 
-@(objc_class="NSApplication")
-Application :: struct {using _: Object}
+ApplicationPresentationOptions :: distinct bit_set[ApplicationPresentationOptionFlag;UInteger]
+ApplicationPresentationOptionsDefault :: ApplicationPresentationOptions{}
+ApplicationPresentationOptionsAutoHideDock :: ApplicationPresentationOptions{.AutoHideDock}
+ApplicationPresentationOptionsHideDock :: ApplicationPresentationOptions{.HideDock}
+ApplicationPresentationOptionsAutoHideMenuBar :: ApplicationPresentationOptions{.AutoHideMenuBar}
+ApplicationPresentationOptionsHideMenuBar :: ApplicationPresentationOptions{.HideMenuBar}
+ApplicationPresentationOptionsDisableAppleMenu :: ApplicationPresentationOptions{.DisableAppleMenu}
+ApplicationPresentationOptionsDisableProcessSwitching :: ApplicationPresentationOptions{
+	.DisableProcessSwitching,
+}
+ApplicationPresentationOptionsDisableForceQuit :: ApplicationPresentationOptions{.DisableForceQuit}
+ApplicationPresentationOptionsDisableSessionTermination :: ApplicationPresentationOptions{
+	.DisableSessionTermination,
+}
+ApplicationPresentationOptionsDisableHideApplication :: ApplicationPresentationOptions{
+	.DisableHideApplication,
+}
+ApplicationPresentationOptionsDisableMenuBarTransparency :: ApplicationPresentationOptions{
+	.DisableMenuBarTransparency,
+}
+ApplicationPresentationOptionsFullScreen :: ApplicationPresentationOptions{.FullScreen}
+ApplicationPresentationOptionsAutoHideToolbar :: ApplicationPresentationOptions{.AutoHideToolbar}
+ApplicationPresentationOptionsDisableCursorLocationAssistance :: ApplicationPresentationOptions{
+	.DisableCursorLocationAssistance,
+}
+
+@(objc_class = "NSApplication")
+Application :: struct {
+	using _: Object,
+}
 
-@(objc_type=Application, objc_name="sharedApplication", objc_is_class_method=true)
+@(objc_type = Application, objc_name = "sharedApplication", objc_is_class_method = true)
```

Comment:
Why are you changing the current style by adding more whitespace and newlines?

Reply:
It's just ols applying autoformatting. I'll make sure to revert all the formatting enforced by it before finishing the draft @gingerBill

Reply:
Yeah, I get it. I'm actually making a new PR with the formatting disabled to keep with the repo styles. Is there any plan to make an `official` language server?